Core/include/Acts/Geometry/Volume.hpp

```diff
@@ -67,20 +63,22 @@ class Volume : public virtual GeometryObject {
   /// returns the center of the volume
   const Vector3& center() const;
 
-  /// returns the volumeBounds()
+  /// Returns const reference to the volume bounds
   const VolumeBounds& volumeBounds() const;
 
   /// Set volume bounds and update volume bounding boxes implicitly
-  void assignVolumeBounds(std::shared_ptr<const VolumeBounds> volbounds);
+  void assignVolumeBounds(std::shared_ptr<VolumeBounds> volbounds);
 
   /// Construct bounding box for this shape
   /// @param envelope Optional envelope to add / subtract from min/max
   /// @return Constructed bounding box pointing to this volume
   BoundingBox boundingBox(const Vector3& envelope = {0, 0, 0}) const;
 
   /// Construct oriented bounding box for this shape
+  /// @note This will build an oriented bounding box with an
+  ///       envelope value of (0.05, 0.05, 0.05)mm
   /// @return Constructed oriented bounding box pointing to this volume
-  const BoundingBox& orientedBoundingBox() const;
+  BoundingBox orientedBoundingBox() const;
```

Comment:
Is the bounding box now created on the fly and not stored?

Reply:
It is, because I wanted to avoid having to also refresh it when volume bounds are reset. It's also a non-zero footprint, that we maybe don't want to store for every volume, but rather in the acceleration structure using it. I would imagine we don't call this function very often.